routes/account.js

```diff
@@ -34,13 +34,16 @@ function accountRoutes (server, options, next) {
     handler: function (request, reply) {
       var username = request.payload.data.attributes.username
       var password = request.payload.data.attributes.password
+      var profile = request.payload.data.attributes.profile
       var id = request.payload.data.id
       var query = request.query
       accounts.add({
         username: username,
         password: password,
-        include: query.include,
```

Comment:
@gr2m is this correct? Or the below? My understanding is that this should be an `option`, not a `property`?